packages/shopify-codemod/README.md

```diff
@@ -12,6 +12,83 @@ This repository contains a collection of Codemods written with [JSCodeshift](htt
 
 ## Included Transforms
 
+### `remove-unused-expressions`
+
+Removes any expressions that don't have any possible side effects.
+
+```sh
+jscodeshift -t shopify-codemods/transforms/remove-unused-expressions <file>
+```
+
+#### Example
+
+```js
+foo;
+!foo;
+123;
+foo.bar;
```

Comment:
Yeah, getters can have side effects, but we don't do that anywhere and ESLint will complain about it anyways.